docs/adr/adr_002_packaging_releasing_admin_UI.md

```diff
@@ -0,0 +1,110 @@
+# Packaging: Upgrading Admin UI
+
+* Status: proposed
+* Deciders: mleplatre, gbeckley, acottner
+* Date: Jan 9, 2024
+
+## Context and Problem Statement
+
+Currently, in order to deliver a change on the Kinto Admin UI to our Remote Settings production users, we need to go through the following steps:
+
+1. Release the Kinto Admin package on NPM
```

Comment:
This isn't done anymore, now we just have to create/tag a release in github and update package.json